utils/generate_types/test/test.ts

```diff
@@ -572,15 +572,6 @@ playwright.chromium.launch().then(async browser => {
   await browser.close();
 })();
 
-(async () => {
-  const browser = await playwright.firefox.launch();
-  const page = await browser.newPage();
-  const context = page.context();
-  const oneTwoThree = ('pageTarget' in context) ? context['pageTarget'] : 123;
```

Comment:
what was the reason that this got removed?

Reply:
Yeah the new typescript doesn't figure this one out, but it wasn't a good test in the first place.

Our typescript tester has been using new typescript instead of old typescript since we upgraded to npm 7. But it will take a separate large patch to fix that.